Consolidation of Mautic Form Types (#632)
mautic/user-documentation: docs/plugins/plugin_resources.rst

* The **Campaign Builder** has the *Push Contact to Integration* action which can be used in the Campaign configuration.

> * **Forms** have the *Push Contact to Integration* action which can be used after a Form is submitted.

Reviewer: Let's use active voice here.

```suggestion
* **Forms** include the **Push Contact to Integration** action, which triggers after a Form submission.
```

Reviewer: @mautic/product-team can you please review if the suggestion here is exactly what it's meant by the PR changes at Mautic core repo? Thanks!
